packages/heading/src/heading.tsx

```diff
@@ -20,11 +20,11 @@ export const Heading: React.FC<Readonly<HeadingProps>> = ({
   ...props
 }) => {
   return (
-    <Slot
```

Comment:
Seems better to remove the Slot from here, but then I think we need to make the component into a `forwardRef` as well, or else the `ref` won't work in this one like it does on the others.

Reply:
oh you're right, adding forwardRef

Reply:
done ✅